studygroups/models.py

```diff
@@ -161,36 +171,45 @@ def accept_application(application):
     application.save()
 
 
-def next_meeting_date(study_group):
-    now = timezone.now()
-    day_delta = list(calendar.day_name).index(study_group.day) - now.weekday()
-    time = study_group.time
-    date = now + datetime.timedelta(days=day_delta)
+def get_all_meeting_times(study_group):
```

Comment:
Haven't gotten round to organize better and I'm still split between the django model way and building more of a data API. What would you suggest?